hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/TestFileSystemStorageStatistics.java

```diff
@@ -74,6 +75,7 @@ public void setup() {
     statistics.incrementBytesReadByDistance(1, RandomUtils.nextInt(0, 100));
     statistics.incrementBytesReadByDistance(3, RandomUtils.nextInt(0, 100));
     statistics.incrementBytesReadErasureCoded(RandomUtils.nextInt(0, 100));
+    statistics.increaseRemoteBytesReadTime(RandomUtils.nextInt(0, 100));
```

Comment:
Can we assert something?

Reply:
The existing test method `testGetLongStatistics` and `testGetLong` applies to any newly added field as well. While running this test locally, I can see assertion results like below

```
2023-04-12 10:37:10,362 INFO  fs.TestFileSystemStorageStatistics (TestFileSystemStorageStatistics.java:testGetLong(99)) - remoteReadTimeMS: FileSystem.Statistics=75, FileSystemStorageStatistics=75

2023-04-12 10:37:10,353 INFO  fs.TestFileSystemStorageStatistics (TestFileSystemStorageStatistics.java:testGetLongStatistics(88)) - remoteReadTimeMS: FileSystem.Statistics=10, FileSystemStorageStatistics=10
```